br/pkg/lightning/mydump/parquet_parser_test.go

```diff
@@ -3,84 +3,85 @@ package mydump
 import (
 	"context"
 	"io"
+	"os"
 	"path/filepath"
 	"strconv"
+	"testing"
 
-	. "github.com/pingcap/check"
 	"github.com/pingcap/tidb/br/pkg/storage"
 	"github.com/pingcap/tidb/types"
+	"github.com/stretchr/testify/assert"
```

Comment:
ditto other PRs. Please keep using `require` consistently.

Reply:
I think sometimes it is reasonable to use `assert`. Now I only use it in tests works like 'table driven' way, when there are many cases independent. It's a good idea to continue the test when one of them failed.